content/popularities.js

```diff
@@ -3,6 +3,7 @@ const path = require("path");
 
 const { CONTENT_ROOT } = require("./constants");
 
-module.exports = JSON.parse(
-  fs.readFileSync(path.join(CONTENT_ROOT, "popularities.json"), "utf-8")
-);
+module.exports = () =>
+  JSON.parse(
+    fs.readFileSync(path.join(CONTENT_ROOT, "popularities.json"), "utf-8")
+  );
```

Comment:
Another idea is to make this one-stop shopping, so have this function take care of both the lazy-loading/caching and lookup, so nobody else has to know the sordid details. So maybe something like:
```js
// Module-level cache
let popularities = null;

module.exports = (url) => {
  if (!popularities) {
    popularities = JSON.parse(
      fs.readFileSync(path.join(CONTENT_ROOT, "popularities.json"), "utf-8")
    );
  }
  return popularities[url];
}
```

and then have all of the importers modify their usage to something like:
```js
const getPopularity = require("./popularities");
...
let popularity = getPopularity(url);
```

Reply:
Won't work because tools like the Flaws Dashboard board need an insight into *all* popularities so it can convert each document's floating point number into a integrate **ranking**. 
But I still don't like the default export and the fact that it's called `popularties` when `getPopularities` would be more appropriate and easier to read.